### ISSUE TITLE
Stay in background

### ISSUE THREAD

**Issue by joetde** [joetde/werigo#13]
Keep the app alive even in background

**Comment by adt-openvibe-x** [comment]
It already seems ok... app working even in backgroud.

**Comment by joetde** [comment]
Yeah, but it is kicked out after some time by Android system. I wonder if there is a way to request the system to don't kill it.
